src/mmw/js/src/modeling/gwlfe/runoff/templates/table.html

```diff
@@ -1,13 +1,13 @@
-<table class="table custom-hover" data-toggle="table">
+<table class="table custom-hover" data-toggle="table" data-height="auto" data-show-footer="true">
```

Comment:
We need `data-height="auto"` otherwise the footer doesn't scroll horizontally with the rest of the table 😞